doc/source/first_simulation.rst

```diff
@@ -0,0 +1,62 @@
+################################
+Launching your first simulation
+################################
+
+The objective of this section is to highlight the steps that are necessary to launch your first simulation after you have cloned and compiled Lethe.
+
+Launching an application requires an executable of the required solver, and a parameters file (with extension .prm if it uses the parameter file format, or .json if it uses the JSON file format). After building Lethe, the solver executable files can be found in : ``$BUILD_FOLDER/applications`` directory.
```

Comment:
```suggestion
Launching an application requires an executable of the required solver, and a parameters file (with extension .prm). After building Lethe, the solver executable files can be found in : ``$BUILD_FOLDER/applications`` directory.
```

We don't support the JSON file format for now so yeah... It would not be that much work, this used to be in development, but we kinda froze it.